src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -43,8 +83,24 @@
   {% set _css_class = _css_class ~ ' ecl-banner--' ~ _size %}
 {% endif %}
 
-{% if _centered %}
-  {% set _css_class = _css_class ~ ' ecl-banner--centered' %}
+{% if _font_size != 'm' %}
+  {% set _css_class = _css_class ~ ' ecl-banner--font-' ~ _font_size %}
+{% endif %}
+
+{% if _font_color != 'dark' %}
+  {% set _css_class = _css_class ~ ' ecl-banner--color-' ~ _font_color %}
+{% endif %}
+
+{% if _background != 'light' %}
```

Comment:
Shouldn't be this two tied together, meaning that if the background is dark then the font is light and viceversa? We request two parameters to handle two options, basically:

-  dark background with light font
- light background with dark font

The choice of a light or dark font should be limited to the transparent background (cool that we added this option, so now people will start complaining because of the text not being readable in random viewports depending on the image or video)

Currently in the stories you can set one of the two resulting in a unreadable text, but i'm saying that this should happen at the template level, imho, not just in the storybook